docs/src/main/paradox/configuration.md

```diff
@@ -49,3 +49,27 @@ Alternatively, Akka Discovery can be used for finding the Cassandra server conta
 in the @extref:[Alpakka Cassandra documentation](alpakka:cassandra.html#using-akka-discovery).
 
 Without any configuration it will use `localhost:9042` as default.
+
+## Using Cassandra-as-a-Service
```

Comment:
we have separate pages for Scylla and CosmosDB, like https://doc.akka.io/docs/akka-persistence-cassandra/current/cosmosdb.html

I think DataStax Astra should be mentioned in same way.

We should probably also structure it with a separate top level index page for these different Cassandra implementations. I can fix this part afterwards.

Reply:
ok, let me know if I should change it.

Reply:
yes, please create a separate page, such as the one for CosmosDB and include in the top level index.md. I can do the adjustments to the structure later.